src/de/arg/arg_action.rs

```diff
@@ -0,0 +1,19 @@
+use clap::ArgAction as AA;
+use serde::Deserialize;
+
+enum_de!(AA, ArgAction,
+    #[derive(Deserialize, Clone, Copy)]
+    #[cfg_attr(feature = "kebab-case-key" ,serde(rename_all = "kebab-case"))]
+    #[cfg_attr(feature = "snake-case-key" ,serde(rename_all = "snake_case"))]
```

Comment:
Either I missed or overlook this before. one dependency activating a feature shouldl't break anerther dependency that doesn't.

There is interest in final artifact (bin, cdynlib, etc) exclusive features.

Reply:
Currently it's a compile error to have duplicate case-key feature.
https://github.com/aobatact/clap-serde/blob/72ff43e84d951e3476b6540b81a309fccfb133c6/src/lib.rs#L8-L20